microsoft/PowerToys
# [FancyZones] Split zones-settings: custom layouts

## Summary of the Pull Request

**What is this about:**

* Keep custom layouts in a separate file.
* Create `custom-layouts.json` from zones-settings on FZ start
* Save custom layouts in the `custom-layout.json` in Editor.
* Unit tests

**What is included in the PR:** 

**How does someone test / validate:** 

* Run PT, verify `custom-layouts.json` created
* Run tests
* Open editor, create or edit the custom layout, verify it was saved properly.

## Quality Checklist

- [x] **Linked issue:** #14782

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
